utils/safernd.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"crypto/rand"
+	"encoding/binary"
+)
+
+func SafeRand() int64 {
+	var safeRndVal int64
+	err := binary.Read(rand.Reader, binary.LittleEndian, &safeRndVal)
+	if err != nil {
+		LavaFormatInfo(err.Error(), nil)
+	}
+	safeRndVal &= 0x7fffffffffffffff // set the highest bit to 0 to get a positive number
+	for safeRndVal == 0 {            // we don't allow 0
```

Comment:
let's remove the denying of 0 to outside this function since other places that will need a stronger rand from crypto/rand might actually allow it